Documentation/contributor-guide/release.md

```diff
@@ -37,48 +37,62 @@ All releases version numbers follow the format of [semantic versioning 2.0.0](ht
+2. Verify you can pass the authentication to the image registries,
+   - `docker login gcr.io`
+   - `docker login quay.io`
+3. Clone the etcd repository and checkout the target branch,
+   - `git clone git@github.com:etcd-io/etcd.git`
+   - `git checkout release-3.X`
+4. Run the release script under the repository's root directory,
+   - `DRY_RUN=false ./scripts/release.sh ${VERSION}`
+
+   It generates all release binaries under directory ./release and images. Binaries are pushed to the google cloud bucket
+   under project `etcd-development`, and images are pushed to `quay.io` and `gcr.io`.
+5. Publish release page in Github
+   - Set release title as the version name
+   - Follow the format of previous release pages
+   - Attach the generated binaries and signature file
+   - Select whether it's a pre-release
+   - Publish the release
+6. Announce to the etcd-dev googlegroup
+
+   Follow the format of previous release emails sent to etcd-dev@googlegroups.com, see an example below,
 ```
-DRY_RUN=false ./scripts/release.sh ${VERSION}
-```
-
-It generates all release binaries and images under directory ./release.
-Binaries are pushed to gcr.io and images are pushed to quay.io and gcr.io.
+Hello,
 
-## Publish release page in GitHub
+etcd v3.4.30 is now public!
 
-- Set release title as the version name.
-- Follow the format of previous release pages.
-- Attach the generated binaries and signatures.
-- Select whether it is a pre-release.
-- Publish the release!
+https://github.com/etcd-io/etcd/releases/tag/v3.4.30
 
-## Announce to the etcd-dev Googlegroup
+Thanks to everyone who contributed to the release!
 
-- Follow the format of [previous release emails](https://groups.google.com/g/etcd-dev).
-- Make sure to include a list of authors that contributed since the previous release - something like the following might be handy:
-
-```
-git log ...${PREV_VERSION} --pretty=format:"%an" | sort | uniq | tr '\n' ',' | sed -e 's#,#, #g' -e 's#, $##'
+etcd team
 ```
-
-- Send email to etcd-dev@googlegroups.com
-
-## Post release
-
-- Create new stable branch through `git push origin ${VERSION_MAJOR}.${VERSION_MINOR}` if this is a major stable release. This assumes `origin` corresponds to "https://github.com/etcd-io/etcd".
-- Bump [hardcoded Version in the repository](https://github.com/etcd-io/etcd/blob/v3.4.15/version/version.go#L30) to the version `${VERSION}+git`.
-
-## Patch release criteria
-
-The etcd project aims to release a new patch version if any of the following conditions are met:
-
-- Fixed one or more major CVEs (>=7.5).
-- Fixed one or more critical bugs.
-- Fixed three or more major bugs.
-- Fixed five or more minor bugs.
+7. Update the changelog to reflect the correct release date.
+8. Paste the release link to the issue raised at step 1 and close the issue.
+9. Crease a new stable branch through `git push origin ${VERSION_MAJOR}.${VERSION_MINOR}` if this is a major stable release.
```

Comment:
I've been meaning to update this doc also, we need to add step to keep our website up to date.
```suggestion
9. Crease a new stable branch through `git push origin ${VERSION_MAJOR}.${VERSION_MINOR}` if this is a major stable release.
10. Update the etcd website to [promote the new release tag](https://github.com/etcd-io/website/blob/main/content/en/docs/v3.5/_index.md?plain=1#L6) for users installing etcd from source.
```

Reply:
Where is the tag being used?  I only see [Install pre-built binaries](https://etcd.io/docs/v3.5/install/#install-pre-built-binaries) mentions the tag. Can we just point users to [releases](https://github.com/etcd-io/etcd/releases) to let users to pickup the latest version?

Reply:
It's used on our install docs currently: https://etcd.io/docs/v3.5/install.

Perhaps we should just fix it in the docs during docs sprint on Tuesday so we don't have to update it there anymore.

Reply:
> It's used on our install docs currently: https://etcd.io/docs/v3.5/install.

But the tag defined in [_index.md](https://github.com/etcd-io/website/blob/168b0ce29e3db94ba97e26b998b36cd6de61e952/content/en/docs/v3.5/_index.md?plain=1#L6) (3.5.12) doesn't the tag used in [install](https://etcd.io/docs/v3.5/install/) (3.5.11). Do you always manually update the installation guide?

So let's not add this (step 10) on this release guide, and only update the installation guide to to point users to [releases](https://github.com/etcd-io/etcd/releases) to let users to pickup the latest version?

Reply:
Updated.

Reply:
Not certain on difference, below is last pr I did for this: https://github.com/etcd-io/website/pull/759/files

Agree let's not update it here and instead change website to just point to releases.